enodi/PostIt-RoR-
# #156966117 Setup test environment

#### What does this PR do?
This PR set up the test environment

#### Description of Task to be completed?
- Setup test environment.
- Write tests for models and ensure they are passing.

#### How should this be manually tested?
- Pull this branch and run `rails s`
- Run `rspec spec/models`
- All tests should pass

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[Story ID: 156966117](https://www.pivotaltracker.com/n/projects/2166405/stories/156966117)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29783644/39082964-9adc3ef4-4553-11e8-811d-29eb5bd28152.png)
